source/javascripts/controllers/_MainController.js.erb

```diff
@@ -277,8 +277,6 @@ angular.module("BitriseWorkflowEditor").controller("MainController", function($s
 		if (originalOnbeforeunloadHandler) {
 			originalOnbeforeunloadHandler(event);
 		}
-
-		window.location.replace();
```

Comment:
@erosdome `location.replace(url)` expects an url to be passed. Everytime I refreshed the browser I got an error, so I removed this I guess it's ok right? If the purpose of this was to trigger the reload: that happens regardless we don't need to do it explicitly